lib/rouge/lexers/solidity.rb

```diff
@@ -62,7 +62,7 @@ def self.constants
       def self.keywords_type
         return @keywords_type if @keywords_type
         @keywords_type = Set.new %w(
-          address bool bytes fixed int ufixed uint string
+          address bool bytes fixed int string ufixed uintb
```

Comment:
I think you had a typo there, `uintb` instead of `uint`.

Reply:
Argh! Sorry—will fix!